src/marqo/tensor_search/tensor_search.py

```diff
@@ -1221,9 +1227,16 @@ def _lexical_search(
     total_preprocess_time = RequestMetricsStore.for_request().stop("search.lexical.processing_before_opensearch")
     logger.debug(f"search (lexical) pre-processing: took {(total_preprocess_time):.3f}ms to process query.")
 
+    max_search_retry_attempts = utils.read_env_vars_and_defaults_ints(EnvVars.MARQO_MAX_BACKEND_SEARCH_RETRY_ATTEMPTS)
+    max_search_retry_backoff = utils.read_env_vars_and_defaults_ints(EnvVars.MARQO_MAX_BACKEND_SEARCH_RETRY_BACKOFF)
```

Comment:
are we sure the backoff values will always be `int`? We may want to accept a backoff of 1.5 seconds, for example.

Reply:
Having floating point backoff values doesn't really matter since it just changes the growth rate of the values by a tiny bit. Keeping it as `int` makes it simple.

Reply:
what happens if an admin tries setting this as a float? a parse error?

Reply:
At least we get this error message, which is readable to a user: https://github.com/marqo-ai/marqo/blob/c607ee3bd8483be0fd7e019a5d28a219977e4093/src/marqo/tensor_search/utils.py#L189

Reply:
yup, since it expects it to be parsed as an int.

Reply:
Yup, `read_env_vars_and_defaults_ints` has some validation and returns a parse error since it expects a string version of an int value